Building for Windows throws "undefined: syscall.Dup2" error

### Expected behavior
When building the application for Windows with bugsnag/panicwrap, the application builds without error.
```
GOOS=windows GOARCH=amd64 go build -o "app.exe"
```

### Observed behavior
The following error is raised
```
vendor/github.com/bugsnag/panicwrap/dup2.go:10:9: undefined: syscall.Dup2
```

### Steps to reproduce
Simply import bugsnag-go or panicwrap.

### Version
bugsnag/panicwrap v1.2.1 and v1.2.2

### Additional information
A recent change (see https://github.com/bugsnag/panicwrap/pull/11/files#diff-d60cb8d026bb18f93c915c1c38a670de9c8163d30ec1258ad95bb02fd30954ebL1) was made that allows dup2.go to be built for Windows. Adding `!windows` to line 1 of dup2.go appears to resolve the issue.

--- a/.github/workflows/test-package.yml
+++ b/.github/workflows/test-package.yml
@@ -5,13 +5,14 @@ on: [ push, pull_request ]
 jobs:
   test:
 
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}-latest
     defaults:
       run:
         working-directory: 'go/src/github.com/bugsnag/panicwrap' # relative to $GITHUB_WORKSPACE
     strategy:
       fail-fast: false
       matrix:
+        os: [ubuntu, windows]
         go-version: ['1.9', '1.10', '1.11', '1.12', '1.13', '1.14', '1.15']
 
     steps:

--- a/.github/workflows/test-package.yml
+++ b/.github/workflows/test-package.yml
@@ -12,23 +12,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: ['1.7', '1.8', '1.9', '1.10', '1.11', '1.12', '1.13', '1.14', '1.15']
+        go-version: ['1.9', '1.10', '1.11', '1.12', '1.13', '1.14', '1.15']
 
     steps:
     - uses: actions/checkout@v2
       with:
         path: 'go/src/github.com/bugsnag/panicwrap' # relative to $GITHUB_WORKSPACE
-    - name: setup go ${{ matrix.go-version }}
-      run: |
-        curl --silent --location --output gimme https://github.com/travis-ci/gimme/raw/v1.5.4/gimme
-        chmod +x ./gimme
-        eval "$(./gimme ${{ matrix.go-version }})"
-        echo "GOOS=" >> $GITHUB_ENV
-        echo "GOARCH=" >> $GITHUB_ENV
-        echo "GOROOT=$GOROOT" >> $GITHUB_ENV
-        echo "PATH=$PATH" >> $GITHUB_ENV
-        echo "GIMME_ENV=$GIMME_ENV" >> $GITHUB_ENV
-        echo "GOPATH=$GITHUB_WORKSPACE/go" >> $GITHUB_ENV
+    - name: set GOPATH
+      run: echo "GOPATH=$GITHUB_WORKSPACE/go" >> $GITHUB_ENV
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
     - name: install dependencies
       run: go get -v -d ./...
     - name: run tests

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,9 @@
+## TBD
+
+### Bug fixes
+
+* Fix windows support by removing undefined syscall
+
 ## 1.3.0 (2021-01-05)
 
 ### Features

--- a/dup2.go
+++ b/dup2.go
@@ -1,4 +1,4 @@
-// +build !arm64
+// +build !arm64,!windows
 
 package panicwrap
 

--- a/panicwrap_monitor_test.go
+++ b/panicwrap_monitor_test.go
@@ -1,0 +1,25 @@
+// +build !windows
+
+package panicwrap
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+)
+
+func TestPanicWrap_monitor(t *testing.T) {
+
+	stdout := new(bytes.Buffer)
+
+	p := helperProcess("panic-monitor")
+	p.Stdout = stdout
+	//p.Stderr = new(bytes.Buffer)
+	if err := p.Run(); err == nil || err.Error() != "exit status 2" {
+		t.Fatalf("err: %s", err)
+	}
+
+	if !strings.Contains(stdout.String(), "wrapped:") {
+		t.Fatalf("didn't wrap: %#v", stdout.String())
+	}
+}

--- a/panicwrap_test.go
+++ b/panicwrap_test.go
@@ -409,22 +409,6 @@ func TestPanicWrap_panicBoundary(t *testing.T) {
 	}
 }
 
-func TestPanicWrap_monitor(t *testing.T) {
-
-	stdout := new(bytes.Buffer)
-
-	p := helperProcess("panic-monitor")
-	p.Stdout = stdout
-	//p.Stderr = new(bytes.Buffer)
-	if err := p.Run(); err == nil || err.Error() != "exit status 2" {
-		t.Fatalf("err: %s", err)
-	}
-
-	if !strings.Contains(stdout.String(), "wrapped:") {
-		t.Fatalf("didn't wrap: %#v", stdout.String())
-	}
-}
-
 func TestWrapped(t *testing.T) {
 	stdout := new(bytes.Buffer)
 

